api/src/main/resources/liquibase-update-to-2.3.xml

```diff
@@ -33,4 +33,16 @@
     	<renameColumn tableName="allergy" oldColumnName="comment" newColumnName="comments" columnDataType="varchar(1024)"/>
     </changeSet>
 
+    <changeSet id="TRUNK-5413" author="odorajonathan" dbms="mysql">
+      <preConditions onFail="MARK_RAN">
+            <not><columnExists tableName="concept" columnName="category"/></not>
+      </preConditions>
+      <comment>
+          Add Category field to OrderSet
+      </comment>
+      <addColumn tableName="concept">
+            <column name="category" type="concept"/>
```

Comment:
Have you seen any existing changeset with this type?

Reply:
I am checking through, though I have not got concept as a type yet

Reply:
Can you check the types for other concept fields in existing changesets?